app/views/includes/modal-free.jade

```diff
@@ -22,6 +22,9 @@ a.modal-action.modal-close.waves-effect.waves-green.btn-flat(title='#{modal.head
                 input#modal_pass.validate(type='password', name='password', required)
                 label(for='modal_pass') #{form.input.pass}
                 p.error.display-none.register-password-error
+            .input-field.col.s12
+                input#checkbox_modal_tos.validate(type='checkbox', name='checkboxTos',required)
```

Comment:
name attrs must not be "camel case", we are using "snake case": https://en.wikipedia.org/wiki/Snake_case